### ISSUE TITLE
SwipeCollectionViewCell doesn't hide

### ISSUE THREAD

**Issue by jayhuo-seekers** [SwipeCellKit/SwipeCellKit#265]
In `Extensions.swipe`, I noticed that the computed property `swipeCells` implementation is different for UICollectionView and UITableView.
```
extension UITableView {
    var swipeCells: [SwipeTableViewCell] {
        return visibleCells.compactMap({ $0 as? SwipeTableViewCell })
    }

    ...
}

extension UICollectionView {
    var swipeCells: [SwipeCollectionViewCell] {
        return visibleCells as? [SwipeCollectionViewCell] ?? []
    }

    ...
}
```

Is it intentional or not? 
I found if there are different kinds of UICollectionViewCells in a UICollectionView which some of them are not subclasses of SwipeCollectionViewCell, the SwipeCollectionViewCells can't hide the actions if I tap somewhere else in the collection view. Because in this case, `swipeCells` are always empty since `visibleCells` now can't be cast into `[SwipeCollectionViewCell]`.

**Comment by kurabi** [comment]
This should be changed to match what UITableView extension does. Please creat a PR.

**Comment by jayhuo-seekers** [comment]
@kurabi Created https://github.com/SwipeCellKit/SwipeCellKit/pull/266

**Comment by kurabi** [comment]
This is now fixed as part of PR #266.
